.github/workflows/black.yml

```
@@ -9,21 +9,29 @@ on:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
       - name: Install black
         run: pip install black
       - name: Run black
         run: black --check --diff --color backend/app
         id: black
+        run: |
```

actionlint (1.7.4)

key "run" is duplicated in element of "steps" section. previously defined at line:23,col:9

(syntax-check)